lk2nd/device/dts/msm8952/msm8952-mtp.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8952 0x0>;
+	qcom,board-id = <QCOM_BOARD_ID_MTP 0>;
+};
+
+&lk2nd {
+    athene {
+        model = "Motorola Moto G4 / G4Plus (athene) (MSM8952)";
+        compatible = "motorola,athene";
+        lk2nd,match-device = "athene*";
```

Comment:
I'd appreciate if we can keep the style consistent across files. lk2nd uses 8-wide hard tabs but this is indented with spaces